src/feature/NavigationBar/constant.js

```diff
@@ -0,0 +1,5 @@
+export const TEXT = {
+  login: "로그인",
+};
```

Comment:
P3:
반복 사용되는 경우에 상수로 관리하는 걸 추천드려요.(검색해보니 로그인이라는 텍스트는 상수로 관리해야할지 잘 모르겠습니다.)
그리고 관습상 상수는 대문자로 작성하시는게 좋습니다.

```suggestion
export const TEXT = {
  LOGIN: "로그인",
};
```